specification/dartLangSpec.tex

```diff
@@ -2571,14 +2581,74 @@ \section{Classes}
 \}
 \end{dartCode}
 
+\LMHash{}%
```

Comment:
This was moved from `\ref{instanceMethods}` to this location, because it is relevant to other members than instance methods as well.